cfme/common/vm_views.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
```

Comment:
Noted. All of these views I checked/tested are common for infra vm's and cloud instances.  I kept those that are only for instance in instances module.